pkg/flow/ent/schema/services.go

```diff
@@ -0,0 +1,36 @@
+package schema
+
+import (
+	"entgo.io/ent"
+	"entgo.io/ent/dialect/entsql"
+	"entgo.io/ent/schema/edge"
+	"entgo.io/ent/schema/field"
+	"entgo.io/ent/schema/index"
+)
+
+// Services holds the schema definition for the Services entity.
+type Services struct {
+	ent.Schema
+}
+
+// Fields of the Services.
+func (Services) Fields() []ent.Field {
+	return []ent.Field{
+		field.String("url").NotEmpty(), // url of the service
```

Comment:
Almost all of our tables have the following columns. They might not be critical, but I don't think there is a cost to adding them either. 

```
field.UUID("id", uuid.UUID{}).Default(uuid.New).Immutable().StorageKey("oid"),
		field.Time("created_at").Default(time.Now).Immutable(),
		field.Time("updated_at").Default(time.Now).UpdateDefault(time.Now),
```